pinot-broker/src/main/java/org/apache/pinot/broker/requesthandler/BaseSingleStageBrokerRequestHandler.java

```diff
@@ -141,18 +141,10 @@ public abstract class BaseSingleStageBrokerRequestHandler extends BaseBrokerRequ
   private static final Expression STAR = RequestUtils.getIdentifierExpression("*");
   private static final int MAX_UNAVAILABLE_SEGMENTS_TO_PRINT_IN_QUERY_EXCEPTION = 10;
   private static final Map<String, String> DISTINCT_MV_COL_FUNCTION_OVERRIDE_MAP = Map.ofEntries(
-      Map.entry("distinctcount", "distinctcountmv"),
```

Comment:
We should not change function override in the same PR. We need to add the function first, then wait for one release before changing the broker override. Otherwise existing queries will fail once we upgrade broker.

We should consider adding a test in the backward compatibility suite

Reply:
Ah whoops, good point, this issue seems to keep coming up recently haha